pkgs/top-level/all-packages.nix

```diff
@@ -33445,9 +33445,13 @@ with pkgs;
   sfxr-qt = libsForQt5.callPackage ../applications/audio/sfxr-qt { };
 
   shadowfox = callPackage ../tools/networking/shadowfox { };
-
+ 
```

Comment:
The newlines around the statement seems to contain whitespaces, what is disliked by the editorconfig checks